StripeConnect/StripeConnect/Source/EmbeddedComponentManager.swift

```diff
@@ -33,6 +33,12 @@ public class EmbeddedComponentManager {
         ComponentAnalyticsClient(client: AnalyticsClientV2.sharedConnect,
                                  commonFields: $0)
     }
+    
+    @_spi(DashboardOnly)
+    public var baseURL: URL = StripeConnectConstants.connectJSBaseURL
+
+    @_spi(DashboardOnly)
+    public var publicKeyOverride: String? = nil
```

Comment:
optional – given that this is required for Dashboard, we could add a `DashboardOnly` convenience init that takes this argument in place of `fetchClientSecret`.